tests/DrupalConsoleStackTest.php

```diff
@@ -0,0 +1,405 @@
+<?php
+use DigipolisGent\Robo\Task\DrupalConsole\DrupalConsoleStack;
+use League\Container\ContainerAwareInterface;
+use League\Container\ContainerAwareTrait;
+use Robo\Common\CommandArguments;
+use Robo\Robo;
+use Robo\TaskAccessor;
+use Symfony\Component\Console\Output\NullOutput;
+
+class DrupalConsoleStackTest extends \PHPUnit_Framework_TestCase implements ContainerAwareInterface
+{
+
+    use DigipolisGent\Robo\Task\DrupalConsole\loadTasks;
+    use TaskAccessor;
+    use ContainerAwareTrait;
+    use CommandArguments;
+
+    /**
+     * Set up the Robo container so that we can create tasks in our tests.
+     */
+    public function setUp()
+    {
+        $container = Robo::createDefaultContainer(null, new NullOutput());
+        $this->setContainer($container);
+    }
+
+    /**
+     * Scaffold the collection builder.
+     *
+     * @return \Robo\Collection\CollectionBuilder
+     *   The collection builder.
+     */
+    public function collectionBuilder()
+    {
+        $emptyRobofile = new \Robo\Tasks;
+        return $this->getContainer()
+            ->get('collectionBuilder', [$emptyRobofile]);
+    }
+
+    /**
+     * Tests a given command.
+     *
+     * @param string $method
+     *   The method to call on DrupalConsoleStack.
+     * @param string $command
+     *   The expected command, without the 'drupal ' prefix and ' --yes' suffix.
+     * @param array $args
+     */
+    protected function doTestCommand($method, $command, array $args = [])
+    {
+        $stack = $this->taskDrupalConsoleStack();
+        call_user_func_array([$stack, $method], $args);
+        $expected = 'drupal ' . $command . ' --yes';
+        $this->assertEquals($expected, $stack->getCommand());
```

Comment:
Don't hide tests in private methods, move the actual check to the `testMethod()`.

```php
protected function doTestCommand($method, $command, array $args = [])
{
    $stack = $this->taskDrupalConsoleStack();
    call_user_func_array([$stack, $method], $args);
    return $stack->getCommand();
}

function testCacheRebuildCommand()
{
    $this->assertEquals(
        'drupal cache:rebuild all --yes',
        $this->doTestCommand('cacheRebuild', 'cache:rebuild ' . static::escape('all'))
    );

$this->assertEquals(
        'drupal cache:rebuild menu --yes',
        $this->doTestCommand('cacheRebuild', 'cache:rebuild ' . static::escape('menu'), ['menu'])
    );
}
```